openrtb_ext/bid.go

```diff
@@ -42,9 +42,9 @@ type BidType string
 
 const (
 	BidTypeBanner BidType = "banner"
-	BidTypeVideo          = "video"
-	BidTypeAudio          = "audio"
-	BidTypeNative         = "native"
+	BidTypeVideo  BidType = "video"
```

Comment:
Was this change necessary? Did the other constants not pick up the correct type?

Reply:
Other than the first one, they all are implicitly being assigned the type "string" rather than BidType. (ie BidTypeVideo = "video" is a string, BidTypeVideo  BidType = "video" is the type BidType)

Definitely necessary, without this examining passing any constant other than the first will generate type errors.